src/entities/client.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    OneToMany,
+} from "typeorm";
+import { Sample } from "@/entities"
+
+@Entity()
+export class Client {
+    @PrimaryGeneratedColumn()
+    id!: number;
+
+    @Column()
+    fullName!: string;
+
+    @Column()
+    email!: string;
+
+    @Column()
+    phoneNumber!: string;
+
+    @Column( { nullable: true } )
+    address?: string;
+
+    @CreateDateColumn()
+    createdAt!: Date;
+
+    @UpdateDateColumn()
+    updatedAt!: Date;
+
+    @OneToMany ( () => Sample, (sample) => sample.client)
+    samples!: Sample[];
+}
+
```

Comment:
deletedat is missing. updatedat should be nullable because the column wont have a value unless you update the row values. move samples  a bit up...above createdat